x-pack/test/fleet_api_integration/apis/space_awareness/enrollment_settings.ts

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../api_integration/ftr_provider_context';
+import { skipIfNoDockerRegistry } from '../../helpers';
+import { SpaceTestApiClient } from './api_helper';
+import { cleanFleetIndices } from './helpers';
+import { setupTestSpaces, TEST_SPACE_1 } from './space_helpers';
+
+export default function (providerContext: FtrProviderContext) {
+  const { getService } = providerContext;
+  const supertest = getService('supertest');
+  const esClient = getService('es');
+  const kibanaServer = getService('kibanaServer');
+  const createFleetAgent = async (agentPolicyId: string, spaceId?: string) => {
+    const agentResponse = await esClient.index({
+      index: '.fleet-agents',
+      refresh: true,
+      body: {
+        access_api_key_id: 'api-key-3',
+        active: true,
+        policy_id: agentPolicyId,
+        policy_revision_idx: 1,
+        last_checkin_status: 'online',
+        type: 'PERMANENT',
+        local_metadata: {
+          host: { hostname: 'host123' },
+          elastic: { agent: { version: '8.15.0' } },
+        },
+        user_provided_metadata: {},
+        enrolled_at: new Date().toISOString(),
+        last_checkin: new Date().toISOString(),
+        tags: ['tag1'],
+        namespaces: spaceId ? [spaceId] : undefined,
+      },
+    });
+
+    return agentResponse._id;
+  };
+  describe('enrollment_settings', async function () {
+    skipIfNoDockerRegistry(providerContext);
+    const apiClient = new SpaceTestApiClient(supertest);
+
+    describe('With Fleet server setup in a specific space', () => {
```

Comment:
If I understood correctly, if there's at least a healthy fleet server across all the spaces, the response should be `has_active: true`. It would be `false` only if there are no healthy fleet servers at all. Is that correct?

Reply:
Yes that is correct

Reply:
Then I'd add a test to cover the case when there are no healthy flee servers. What do you think?

Reply:
Just added a test